lib/adapter.js

```diff
@@ -7000,7 +6995,7 @@ function Adapter(options) {
                     // just read one alias Object
                     adapterObjects.getObject(pattern, options, (err, aliasObj) => {
                         err && logger.warn(`${this.namespaceLog} cannot read ${pattern}: ${err}`);
-                        aliasObj && this._addAliasSubscribe(aliasObj, pattern, callback);
+                        aliasObj ? this._addAliasSubscribe(aliasObj, pattern, callback) : typeof callback === 'function' && callback();
```

Comment:
At the risk of going on your nerve:
Can we not please not misuse ternaries aswell for control flow? That's what if/else is for.
`condition && action()` is somewhat ok (I still hate it), but this is just unreadable.

Reply:
everything fine.. ;-) just wanted to add the missing cb execution for some cases in a way most similar to the existing one ;-)

Reply:
Maybe its possible to define it in eslint, otherwise this style will most likely be reapplied by someone